csuntechlab/iSTART
# STR-187 -- Unify typography within the modules.

# Description
Unified module typography, Refactored re-usable classes, Added margin.

# Testing
1. Ensure typography looks good on all devices.
2. Ensure proper margins are added for header, sub-header, text, and captions.
3. Make sure typography is consistent with all templates.
